src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -535,6 +543,8 @@ public void periodic() {
 
     actualXPositionLog.log(armXPosition);
     actualYPositionLog.log(armYPosition);
+    SmartDashboard.putNumber("Inverse Elbow", Units.radiansToDegrees(convertToElbowAngle(armXPosition, armYPosition)));
```

Comment:
Why is this added?  Seems like we should have the inverse in the actualAngle, and that this is debugging code?

Reply:
removed